applications/EigenSolversApplication/CMakeLists.txt

```diff
@@ -22,6 +22,7 @@ if( NOT MSVC )
 endif()
 
 add_definitions( -DEIGEN_DEFAULT_TO_ROW_MAJOR -DEIGEN_MPL2_ONLY )
+#add_definitions( -DEIGEN_INTERNAL_DEBUGGING )
```

Comment:
@armingeiser maybe it makes sense to enable this when Kratos is compiled in `FullDebug`?

Reply:
i think its a good idea, except i don't know how to detect if Kratos is compiled in FullDebug.

Reply:
@armingeiser From CMake files, you need to check if CMAKE_BUILD_TYPE is "FullDebug"

Reply:
You can use:

```CMake
if( ${CMAKE_BUILD_TYPE} MATCHES "FullDebug" )
...
endif( ${CMAKE_BUILD_TYPE} MATCHES "FullDebug" )
```

Reply:
thanks!